Pipelines/Scripts/repackage-for-release.ps1

```diff
@@ -0,0 +1,130 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+<#
+.SYNOPSIS
+    Repackages release candidate tgz to  the Mixed Reality Toolkit Unity Package Manager (UPM) packages.
+.DESCRIPTION
+    Builds UPM packages for the Mixed Reality Toolkit.
+.PARAMETER PackageRoot
+    The root folder of the project.
+.PARAMETER OutputDirectory
+    Where should we place the output? Defaults to ".\artifacts"
+.PARAMETER BuildNumber
+    The fourth digit (revision) for the full version.
+.PARAMETER PreviewTag
+    The tag to append after the version, including the preview number (e.g. "internal.0" or "pre.100")
+.PARAMETER Revision
+    The revision number for the build.  (e.g. "230508.3" or "2") 
+.PARAMETER ReleasePackages
+    An array of the package names that are approved for release.  If the package isn't in this array, it will get labeled -pre
+#>
+param(
+    [Parameter(Mandatory = $true)]
+    [string]$PackageRoot,
+
+    [string]$OutputDirectory = "./artifacts/upm",
+
+    [ValidatePattern("\d+\.\d+\.\d+")]
+    [string]$Version,
```

Comment:
Since the release packages come from the "prerelease" release candidates, can we just reuse the version information in those packages, and remove the ".prerelease" and "build iteration" numbers.  Then I think you can remove a lot of the script inputs.  Thoughts?
 
For exmaple. 

**Input Version Info**
 [Major].[Minor].[Patch]-prerelease.pre.[PreviewNumber].[DateNumber].[Counter] 
 
 _regex to remove "-prerelease.pre.[PreviewNumber].[DateNumber].[Counter]" and re-add "-pre.[PreviewNumber]"_
 
 **Output**
 [Major].[Minor].[Patch]-pre.[PreviewNumber]
 
 Or 

**Input Version Info**
 [Major].[Minor].[Patch]-prerelease.[DateNumber].[Counter] 
 
_regex to remove "-prerelease.[DateNumber].[Counter]"_
 
 **Output**
 [Major].[Minor].[Patch]